go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/openshift/installer
 go 1.14
 
 require (
-	cloud.google.com/go v0.57.0
+	cloud.google.com/go v0.65.0
```

Comment:
This seems to be tightly coupled with our GCP provider.  I'm unable to update our provider and pin this simultaneously.

https://github.com/openshift/installer/pull/4396/files#r527192462